testing_and_setup/compass/ocean/global_ocean/EC60to30JS/init/define_base_mesh.py

```diff
@@ -0,0 +1,10 @@
+#/usr/bin/env python
+
+import mesh_definition_tools as mdt
+
+def cellWidthVsLatLon():
+    params = mdt.params
+    lon,lat,cell_width = mdt.create_background_mesh(params["grd_box"],params["ddeg"],params["mesh_type"],params["dx_max"])
+    cell_width = cell_width/1000
```

Comment:
May want to make this 1000.0 just to be clear it is a float (and to avoid integer division issues).